interface/orders/orders_results.php

```diff
@@ -392,25 +392,25 @@ function validate(f) {
 
  <tr class='head'>
   <td colspan='2'><?php echo $form_batch ? xl('Patient') : xl('Order'); ?></td>
-  <td colspan='4'><?php xl('Report', 'e'); ?></td>
-  <td colspan='7'><?php xl('Results and', 'e'); ?> <span class='reccolor''>
-    <?php  xl('Recommendations', 'e'); ?></span></td>
+  <td colspan='4'><?php xlt('Report', 'e'); ?></td>
+  <td colspan='7'><?php xlt('Results and', 'e'); ?> <span class='reccolor''>
+    <?php  xlt('Recommendations', 'e'); ?></span></td>
  </tr>
 
  <tr class='head'>
-  <td><?php echo $form_batch ? xl('Name') : xl('Date'); ?></td>
-  <td><?php echo $form_batch ? xl('ID') : xl('Procedure Name'); ?></td>
-  <td><?php xl('Reported', 'e'); ?></td>
-  <td><?php xl('Ext Time Collected', 'e'); ?></td>
-  <td><?php xl('Specimen', 'e'); ?></td>
-  <td><?php xl('Status', 'e'); ?></td>
-  <td><?php xl('Code', 'e'); ?></td>
-  <td><?php xl('Name', 'e'); ?></td>
-  <td><?php xl('Abn', 'e'); ?></td>
-  <td><?php xl('Value', 'e'); ?></td>
-  <td><?php xl('Units', 'e'); ?></td>
-  <td><?php xl('Range', 'e'); ?></td>
-  <td><?php xl('?', 'e'); ?></td>
+  <td><?php echo $form_batch ? xlt('Name') : xlt('Date'); ?></td>
+  <td><?php echo $form_batch ? xlt('ID') : xlt('Procedure Name'); ?></td>
+  <td><?php xlt('Reported', 'e'); ?></td>
+  <td><?php xlt('Ext Time Collected', 'e'); ?></td>
+  <td><?php xlt('Specimen', 'e'); ?></td>
+  <td><?php xlt('Status', 'e'); ?></td>
+  <td><?php xlt('Code', 'e'); ?></td>
+  <td><?php xlt('Name', 'e'); ?></td>
+  <td><?php xlt('Abn', 'e'); ?></td>
+  <td><?php xlt('Value', 'e'); ?></td>
+  <td><?php xlt('Units', 'e'); ?></td>
+  <td><?php xlt('Range', 'e'); ?></td>
+  <td><?php xlt('?', 'e'); ?></td>
```

Comment:
For the above 11 lines:
Correct use of xlt(), but need to remove the `e` parameter and add the `echo`